test_pytest_html.py

```diff
@@ -37,6 +48,7 @@ def assert_summary(html, tests=1, duration=None, passed=1, skipped=0, failed=0,
     assert int(re.search('(\d)+ errors', html).group(1)) == errors
     assert int(re.search('(\d)+ expected failures', html).group(1)) == xfailed
     assert int(re.search('(\d)+ unexpected passes', html).group(1)) == xpassed
+    assert_summary_checkboxes(html, passed)
```

Comment:
Rather than call another method and only check passed, it wouldn't be much additional code to check all boxes are in the HTML and have the appropriate state. You could potentially use `**kwargs` to loop over the outcomes passed to this method and reduce duplication. You might then need a dict for mapping `xfailed` to `expected failures`, etc.